node/resolvers/Mutations/Users.ts

```diff
@@ -123,6 +133,14 @@ export const addUser = async (_: any, params: any, ctx: Context) => {
   } = ctx
 
   try {
+    const userExists = await getUser({ masterdata, params })
+
+    if (userExists?.orgId !== params?.orgId) {
+      throw new Error(
+        `User with email ${params.email} and orgId ${params.orgId} already exists in another org.`
```

Comment:
I think we should throw an error even if they already exist in the same org. Like, if you're trying to add a user as admin but they already exist in the same org as a buyer, the app should throw an error. The correct operation in that case would be to edit the existing user.